RedBugs_webapp/src/main/java/com/isa/pl/redbugs/service/RouteService.java

```diff
@@ -0,0 +1,25 @@
+package com.isa.pl.redbugs.service;
+
+import com.isa.pl.redbugs.model.Route;
+
+import java.util.List;
+
+public class RouteService {
+
+    public Route createRoute(long routeId, long numberVehicle, String nameRoute, long[] stops) {
+        return new Route(routeId, numberVehicle, nameRoute, stops); // should the created route be added to list in InitData?
```

Comment:
```suggestion
    public void createRoute(Route route) {
        return writerJson(route)
```
przy tworzeniu obiektu w parametrze obiektu przekazujemy cały obiekt.

Reply:
Commits with updates pushed